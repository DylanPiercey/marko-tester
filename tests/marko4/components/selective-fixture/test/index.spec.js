const { fixtures } = require('../../../../../src/index')('../index.marko', { withoutFixtures: true });

fixtures('hidden');
