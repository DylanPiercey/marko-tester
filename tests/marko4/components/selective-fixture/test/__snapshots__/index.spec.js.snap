// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component with hidden fixture 1`] = `
Array [
  ,
  ,
]
`;
